react_context_lab_app/src/App.css

```diff
@@ -0,0 +1,77 @@
+#App {
+  text-align: center;
+  /* background-color: var(--color-background); */
+
+  /* --color-background: #fcfcfc;
+  --color-containers: #ededed;
+  --color-hover: #5f8894;
+  --color-navbarActive: #fb3f3f;
+  --color-text: #080808; */
+}
+
+.light {
+  text-align: center;
+  background-color: var(--color-background);
```

Comment:
usually for themes you would want to have variables like `primary-color`, `secondary-color`, `tertiary-color`